platform/broadcom/sai.mk

```diff
@@ -1,4 +1,4 @@
-LIBSAIBCM_XGS_VERSION = 10.1.20.0
+LIBSAIBCM_XGS_VERSION = 10.1.35.0
 LIBSAIBCM_DNX_VERSION = 7.1.111.1
 LIBSAIBCM_XGS_BRANCH_NAME = SAI_10.1.20.0
```

Comment:
LIBSAIBCM_XGS_BRANCH_NAME should be 'SAI_10.1.0_GA'

Reply:
Thanks @Gfrom2016 ! @kperumalbfn as FYI. : D